editor/src/components/editor/npm-dependency/npm-dependency.ts

```diff
@@ -324,6 +324,26 @@ export function dependenciesFromPackageJson(
   }
 }
 
+export function includesDependency(
+  packageJsonFile: ProjectFile | null,
+  dependencyToCheck: string,
+): boolean {
+  if (packageJsonFile != null) {
+    if (isTextFile(packageJsonFile)) {
+      const parsedJSON = json5.parse(packageJsonFile.fileContents.code)
+      const fromDependencies = Utils.path<unknown>(['dependencies', dependencyToCheck], parsedJSON)
+      const fromDevDependencies = Utils.path<unknown>(
+        ['devDependencies', dependencyToCheck],
+        parsedJSON,
+      )
```

Comment:
I've created the follow up ticket #1468 for this, because we need further support for `devDependencies`